app/scripts/directives/pageMessage.dir.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+/*
+  To use the pageMessage function, call the PageMessage Service from the state's controller.
+*/
+
+angular.module('yourStlCourts').directive('pageMessage', function($rootScope){
+  return {
+    restrict: 'E',
+    templateUrl: 'views/pageMessage.html',
+    controller: 'PageMessageCtrl as ctrl',
+    link: function ($scope, element, attrs) {
```

Comment:
Ok the directive is fine but you really should just be able to use ng-class for your `page-message-no-show` toggling in `pageMessage.html`